### ISSUE TITLE
Document the deploy setup, clean up github actions

### ISSUE THREAD

**Issue by bmann** [filecoin-project/FIPs#158]
Hi @andyschwab can you please document the deploy process, either here in this issue or perhaps in the [WEBSITE.md file](https://github.com/filecoin-project/FIPs/blob/master/WEBSITE.md)

It looks like you are using Fleek rather than using the Fission github action I had commited. No problem, if that's what you're more familiar with, but I wish you had actually talked to me about it in the issues publicly first, rather than just doing it.

I looked at the docs, and it looks like Fleek doesn't support a deploy on PR, so that PR submitters can get a preview. This would be very useful to have, can you please confirm whether this works or not? 

I will make a PR removing the Fission github action. It would be ideal to have the Fleek info committed to the repo so everyone knows how it works. Can you take that on @andyschwab?

My plan is to use the forked repo at https://github.com/application-research/FIPs to work on design to not clutter up the working FIP repo here, and PR in the changes as needed. Everyone welcome there!

**Comment by kaitlin-beegle** [comment]
Hey @bmann! Since you have a forked FIPs repo for the FIPs website, I'm going to transfer this issue over there.  

Also, you may want to connect with Andy offline and then link any documentation.

**Comment by bmann** [comment]
@kaitlin-beegle the deployment is done using this canonical repo as the base. Everything for deploying / working with this repo, should be kept in here.

I'm trying to get baseline stuff documented. Including whether or not _I_ should remove the extraneous Fission deployment from this repo, or @andyschwab is going to do it.

And especially clarity around per-PR deployment support, because I'd like to have FIP submitters have their own web previews so they know what it's going to look like, and can keep updating their own PRs.

**Comment by andyschwab** [comment]
@bmann - thanks for your patience! 

The current deploy process is integrated with Fleek.co, which handles the bulk of the process. Right now, the configuration is managed directly through Fleek, but we can also add a [.fleek.json](https://docs.fleek.co/internet-computer-hosting/site-deployment/#file-configuration) file to enable in-repo deployment management if necessary.

Per your request, I've setup PRs to main to generate a preview link as well.

I think the critical thing right now is getting the architecture reverted per jbenet's request. Once that is complete, we can look at how best to keep fips.filecoin.io live without impacting the user experience.

**Comment by bmann** [comment]
@andyschwab Great! Yes, I think per PR builds are crucial, please do set that up.

Will you remove the Github actions for Fission deploy or should I?

If the fleek json has all the details, then it's good for discoverability / documentation when others are looking at it or forking it.

**Comment by kaitlin-beegle** [comment]
Thanks @bmann and @andyschwab for helping to resolve this! I'm going to close this issue since a new PR has been submitted.
